realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -1026,7 +868,7 @@ public static String getTableNameForClass(String name) {
         return TABLE_PREFIX + name;
     }
 
-    protected native long createNative();
+    static native long nativeCreate();
```

Comment:
Good catch!
`Table.pivot()` was the only user of this method, but I removed it!